lambdas/FetchJobsData.py

```diff
@@ -0,0 +1,62 @@
+import psycopg2
+import json
+import os
+import logging
+
+try:
+    # for local development, load environment variables from .env file
+    import dotenv
+    dotenv.load_dotenv()
+except ImportError:
+    logging.warning('dotenv not available, assuming running in AWS environment')
+
+def get_db_connection():
+    try:
+        return psycopg2.connect(
+            host=os.environ['DB_HOST'],
+            port=os.environ['DB_PORT'],
+            dbname=os.environ['DB_NAME'],
+            user=os.environ['DB_USER'],
+            password=os.environ['DB_PASSWORD'])
+    except Exception as e:
+        logging.error(f"Error connecting to the database: {e}")
+        raise
+
+def fetch_data(cursor):
+    try:
+        cursor.execute("SELECT id, title, city, location, company, job_type, date_posted, job_url FROM jobs ORDER BY date_posted DESC")
+        # Fetch the column names
+        colnames = [desc[0] for desc in cursor.description]
+        # Use a list comprehension and zip to create a list of dictionaries
+        return [dict(zip(colnames, row)) for row in cursor.fetchall()]
```

Comment:
There is no need for excessive comments. Aim to make the code simple to understand

Your code should be self-documented.

Please watch the following video: [link](https://www.youtube.com/watch?v=Bf7vDBBOBUA)